### PR TITLE
Local Conda Setup

### PR DESCRIPTION
**Pull request by brittainhard**

### CODE REVIEW

**Comment by brittainhard** [comment]
@ahmadia Can you test this? You will still have to do the step of copying `source/memex/settings_files/dev_settings.py` to `source/memex/settings.py` and running `python manage.py migrate`. After that, run `supervisord` from the source directory. This will all be added to the readme.

**Comment by brittainhard** [comment]
Merging if no comment. Review appreciated.

File: source/app_setup.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+pushd ../
+conda env update --file local-environment.yml;
```
Comment (ahmadia) [inline source/app_setup.sh]: This only works if the user has conda env installed and the memex environment activated.  You need:

(install latest version of conda if it's not installed yet)
then:  `conda env create --name memex --file /vagrant/environment.yml`

**Comment by ahmadia** [comment]
@brittainhard - I'll drop issues into this thread as I run into them.

**Comment by brittainhard** [comment]
how about this? 
```
-conda env update --file local-environment.yml;
+conda update conda;
+conda install conda-env -y;
+conda env create --file local-environment.yml;
```

**Comment by ahmadia** [comment]
That should be fine.  I think conda-env is available in the standard ways conda is installed.  I got confused because it looks like there are two environment files?  

tika and elasticsearch are not coming up.  We're looking into it.

**Comment by brittainhard** [comment]
what kind of machine is he on

**Comment by ahmadia** [comment]
No Java installed.

**Comment by ahmadia** [comment]
everything's up

**Comment by ahmadia** [comment]
fantastic f*cking work Brittain.  Everything's up.

**Comment by brittainhard** [comment]
awesome

**Comment by ahmadia** [comment]
testing ache now.  This is on a Linux box.

**Comment by chrismattmann** [comment]
Love the energy on this fan fuckin tastic!

**Pull request merged**